### ISSUE TITLE
Trim line compare window... 

### ISSUE THREAD

**Issue by bvahedy** [AndBible/and-bible#1844]
User requests these features to make the compare window more tidy:

https://support.andbible.org/scp/tickets.php?id=895

There's something I can't do, but what I would do (if I could) - I would minimize the empty space:
a) between the version abbreviation (e.g. ESV2011) and the Bible verse;
b) between Bible verse and a line
c) between line and version abbreviation.

**Comment by agrogers** [comment]
Yes, I agree with that too. Bit too much padding for my liking. I can work
on that.

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by bvahedy** [comment]
That would be great Andrew, thank you.
